docs/what-is-tizen/versions/tizen-4-0-m1.md

```diff
@@ -1,64 +1,60 @@
 # Tizen 4.0 M1
 
-The [first milestone (M1) of Tizen 4.0 was released](../../open-source-tizen/release-notes/tizen-4-0-m1.md) in May 2017, marking a new turning point for the platform, which is currently the most successful Linux-based embedded OS in the world.
-
-
-
-The key changes in Tizen 4.0 are optimizations made on the platform for IoT developers to enable rapid creation and commercialization of various applications. While the existing Tizen platform was limited in distribution to devices such as TVs and smartphones, Tizen 4.0 will provide a development environment that can be refined according to the characteristics of various devices by subdividing functional modules. In addition, the Tizen 4.0 platform has been extended to Tizen RT (Real-Time) to involve high-end products such as TVs and mobile devices as well as low-end products such as thermostats, scales, bulbs, and more.
+The first milestone (M1) of Tizen 4.0 was released in May 2017, marking a new turning point for the platform, which is currently the most successful Linux-based embedded OS in the world.
 
+The key changes in Tizen 4.0 are optimizations made on the platform for IoT developers to enable rapid creation and commercialization of various applications. While the existing Tizen platform was limited in distribution to specific devices, such as TVs and smart phones, Tizen 4.0 provides a development environment that can be refined according to the characteristics of various devices by subdividing functional modules. In addition, the Tizen 4.0 platform has been extended to Tizen RT (Real-Time) to involve both high-end products (such as TVs and mobile devices) and low-end products (such as thermostats, scales, and bulbs).
 
+For more information, see the [Tizen 4.0 M1 release note](../../open-source-tizen/release-notes/tizen-4-0-m1.md).
 
 ![Tizen 4.0 M1 announcement](media/4.0_m1_announcement.png)
 
+## New Development Experience for a Strengthened Application Ecosystem
 
+While many existing OS platforms are limiting due to their framework, policies, or business models, Tizen 4.0 aims to promote innovation and expansion of the application ecosystem by giving developers more freedom and flexibility.
 
+One way the platform accomplishes this is through the introduction of .NET. With .NET, you can enjoy C#, the familiar, industry-leading programming language and one of the richest standard libraries.
 
+Additionally, Tizen Open Source Project worked closely with Microsoft to develop [Visual Studio Tools for Tizen](https://news.samsung.com/global/samsungs-third-tizen-net-developer-preview-introduces-new-visual-studio-tools), which are integrated into Visual Studio, a popular IDE (Integrated Development Environment), and Xamarin.Forms, a cross-platform UI toolkit. Because Xamarin.Forms supports cross platforms, you can build native UIs for Android, iOS, and Windows from a single, shared C# code base, and easily port these applications to Tizen.
```

Comment:
The link leads to some news site: https://news.samsung.com/global/samsungs-third-tizen-net-developer-preview-introduces-new-visual-studio-tools.
Could this be modified to lead to the Visual Studio Tools for Tizen section in Tizen Developers (https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen)?

Reply:
developer.tizen.org will be closed when the new portal is officially opened. 
So... the current link is correct.

Reply:
OK